evansmusomi/burger-builder
# Refactors App to Use Redux for Ingredient and Price State

*What this does*
- Uses redux instead of local state for referencing and manipulating ingredients and price info

*How to verify*
- User can complete the build, checkout, order flow without seeing query params
- Developer can verify the mapping of state to props and dispatch to props
